src/elements/common/content-answers/ContentAnswersModal.tsx

```diff
@@ -180,16 +178,15 @@ const ContentAnswersModal = ({
             isMarkdownEnabled={isMarkdownEnabled}
             isResetChatEnabled={isResetChatEnabled}
             onClearAction={handleClearConversation}
-            onModalClose={handleOnRequestClose}
             onOpenChange={handleOnRequestClose}
             open={isOpen}
             questions={questions}
             retryQuestion={handleRetry}
+            setAnswerFeedback={undefined}
```

Comment:
This failed the lint step if we didn't declare a value. For the moment, we pass undefined in order to not show the feedback yet